src/zkbob/manager/AllowListOperatorManager.sol

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: CC0-1.0
+
+pragma solidity 0.8.15;
+
+import {IOperatorManager} from "../../interfaces/IOperatorManager.sol";
+import {Ownable} from "../../utils/Ownable.sol";
+
+/**
+ * @title AllowListOperatorManager
+ * @dev Implements an allow list based access control for ZkBobPool relayers.
+ */
+contract AllowListOperatorManager is IOperatorManager, Ownable {
+    // if true, only whitelisted addresses can be operators
+    // if false, anyone can be an operator
+    bool public allowListEnabled;
+
+    // mapping of whitelisted operator addresses
+    mapping(address => bool) public operators;
+
+    // mapping of fee receivers for operators
+    mapping(address => address) public operatorFeeReceiver;
```

Comment:
I think we can keep only one mapping here with fee receivers, and don't occupy more storage slots with bool flags.

Reply:
Agree, I'll remove operators mapping if we decide that the issue described above isn't particularly significant.

Reply:
I guess it is not possible if we want to allow fee receivers to claim fees after the operator has been disabled.

Reply:
We can pack it in a single struct of `address` and `bool`, compiler will put them in a single storage slot in such case

Reply:
Done in https://github.com/zkBob/zkbob-contracts/pull/97/commits/8d193842f3836649ddea99afce13e6d34225cfcb